scipy/stats/_axis_nan_policy.py

```diff
@@ -254,7 +260,8 @@ def axis_nan_policy_wrapper(*args, _no_deco=False, **kwds):
                 # currently the hypothesis tests this is applied to do not
                 # propagate nans in a sensible way
                 if any(contains_nans) and nan_policy == 'propagate':
-                    return result_object(np.nan, np.nan)
+                    res = np.ones(n_outputs) * np.nan
```

Comment:
A really small thing here but this could be simplified using [np.full](https://numpy.org/doc/stable/reference/generated/numpy.full.html) e.g. ``res = np.full(n_outputs, np.nan)``

Reply:
Thanks, actually. I've not gotten in the habit of using that but i like it.

Reply:
```suggestion
                    res = np.full(n_outputs, np.nan)
```